resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -141,6 +141,8 @@ lambda.function.updateCode.action=Update Function Code
 lambda.function.delete.action=Delete Function
 lambda.application.delete.action=Delete Application
 serverless.application.deploy=Deploy Serverless Application
+sam.executable.version_parse_error=Could not parse SAM executable version from "{0}"
+sam.executable.version_wrong=Bad SAM executable version. Expected {0} <= version < {1} but was {2}
```

Comment:
Can we be more specific here - I'd like to give customers a specific resolution path.

If it's less than the required version, then they need to upgrade SAM to at least the minimum version.
If it's greater than the accepted range, then they need to upgrade their AWS Toolkit plugin.

Reply:
done